libraries/AP_Notify/ToneAlarm.cpp

```diff
@@ -93,6 +93,8 @@ const AP_ToneAlarm::Tone AP_ToneAlarm::_tones[] {
     { "MFMST200L64O4ceceP32ceceP8df#df#P32df#df#P8L16gf#g>c", false },
 #define AP_NOTIFY_TONE_NO_SDCARD 30
     { "MNBGG", false },
+#define AP_NOTIFY_TONE_FLIGHT_PLAN_LOAD 31
+    { "MFT220 ML O3ef O4c", true },
```

Comment:
the 'true' means continuous - is that really wanted? I would have thought a one-off play of the tune would be what you would want

Reply:
I misinterpreted the meaning of "continuous" -- I will change this to false. One-off is what I want